From 6186d9cec9d353ea537e179c23978c4fe7577017 Mon Sep 17 00:00:00 2001
From: Roman Stratiienko <r.stratiienko@gmail.com>
Date: Thu, 19 Oct 2023 03:19:47 +0300
Subject: [PATCH 11/12] base: Set TARGET_FLATTEN_APEX to true

Android-14 requires either TARGET_FLATTEN_APEX to be set to true or
ro.apex.updatable sysprop set to true. Otherwise, init will enter a
boot loop.

Signed-off-by: Roman Stratiienko <r.stratiienko@gmail.com>
---
 common/base/device.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/common/base/device.mk b/common/base/device.mk
index d0210bf..c086dfe 100644
--- a/common/base/device.mk
+++ b/common/base/device.mk
@@ -74,3 +74,5 @@ PRODUCT_PACKAGES += \
     update_engine_client \
     update_verifier \
     update_engine_sideload \
+
+TARGET_FLATTEN_APEX := true
-- 
2.39.2

